src/tests/data/pubkey/ml_kem.vec

```diff
@@ -1,203 +1,203 @@
-# This file was auto-generated from the KATs at https://github.com/post-quantum-cryptography/KAT/tree/main/MLKEM
+# This file was auto-generated from the reference implemention's KATs
```

Comment:
This change stems from the `gen_kyber_kat.py` but is not correct since these KATs are from the `post-quantum-cryptography/KAT` repository and not directly from the reference implementation.